sheepdog/utils/transforms/graph_to_doc.py

```diff
@@ -153,6 +153,8 @@ def list_to_comma_string(val):
     """
 
     if val is None:
+        if file_format == "tsv":
+            return ""
         return val
```

Comment:
i think this block should be refactored to be `if file format is JSON: return val; else: return ""` because we would probably expect CSV format to be similar to TSV format